config/application.rb

```diff
@@ -43,5 +43,13 @@ class Application < Rails::Application
     # Middleware like session, flash, cookies can be added back manually.
     # Skip views, helpers and assets when generating a new resource.
     config.api_only = true
+
+    # This also configures session_options for use below
+    config.session_store :cookie_store, key: ENV.fetch("SECRET_KEY_BASE")
```

Comment:
Will this also need to go into local setup `env.sample` now it is used explicitly